src/test/java/jpa/domain/member/MemberRepositoryTest.java

```diff
@@ -47,15 +47,9 @@ void setUp() {
         memberRepository.save(member1);
         memberRepository.save(member2);
 
-        Station gangnam_station = stationRepository.save(Station.builder()
-                .name("강남역")
-                .build());
-        Station jamsil_station = stationRepository.save(Station.builder()
-                .name("잠실역")
-                .build());
-        Station pangyo_station = stationRepository.save(Station.builder()
-                .name("판교역")
-                .build());
+        Station gangnam_station = stationRepository.save(Station.of("강남역"));
+        Station jamsil_station = stationRepository.save(Station.of("잠실역"));
+        Station pangyo_station = stationRepository.save(Station.of("판교역"));
```

Comment:
지금 확인했는데 변수명 네이밍 컨벤션이 맞지 않네요
자바의 경우 카멜케이스로 아래와 같이 작성해주셔야 합니다.

```suggestion
        Station gangnamStation = stationRepository.save(Station.of("강남역"));
        Station jamsilStation = stationRepository.save(Station.of("잠실역"));
        Station pangyoStation = stationRepository.save(Station.of("판교역"));
```

테스트고 한글로 작성하는 케이스도 있어 꼭 지키지는 않아도 되겠지만, 한글이나 static 변수가 아니라면 가급적 컨벤션을 지켜주시면 좋을 것 같아요

Reply:
앗😵  테스트에도 컨벤션을 지키도록 하겠습니다!